### ISSUE TITLE
Migrate: Postgres connection string without port prints `undefined`

### ISSUE THREAD

**Issue by timsuchanek** [prisma/prisma#5429]
I have a Postgres connection string of this form:
```
postgres://postgres:pw@lol.hello.eu-west-1.rds.amazonaws.com/my-db
```

When I do `prisma db push`, I get this:
```
Datasource "db": PostgreSQL database "my-db", schema "public" at "lol.hello.eu-west-1.rds.amazonaws.com:undefined"
```
It seems like we should fill in the default value for the port if it's not in the connection string.

**Comment by ksuh90** [comment]
I am bumping into the same issue here. I'm using a third party deployment service where the internal domain of my postgres instance does not contain the port number. After upgrading to 2.16, the prisma instance is unable to connect with the same error message above.

**Comment by Jolg42** [comment]
@ksuh90 the issue here is actually about a display issue, connection works.

If your connection url domain is `localhost`, you could try `2.16.1` just released now that fixes it:
See https://github.com/prisma/prisma/releases/tag/2.16.1

If that doesn't fix it, open a new issue so we can try to reproduce 😃

**Comment by janpio** [comment]
The fix in 2.16.1 was unfortunately not for Postgres, so if you are experiencing problems with Postgres and `localhost`, please open a new issue @ksuh90 and we are happy to look into that.
